src/rules/no-redundant-files.ts

```diff
@@ -13,8 +13,14 @@ const defaultFiles = [
 	/^(\.\/)?package\.json$/i,
 ];
 
+const hasWildcards = (filename: string): boolean => /[*?[\]{}]/.test(filename);
```

Comment:
Actually, I do have one request.  Can we move this `regex` outside of the function as a `const`?  There's nothing dynamic about it, and it would prevent us from having to create a new instance every function call.

Reply:
Done